crates/editor/src/display_map/wrap_map.rs

```diff
@@ -1019,10 +1019,11 @@ impl Iterator for WrapRows<'_> {
         Some(if soft_wrapped {
             RowInfo {
                 buffer_id: None,
-                buffer_row: None,
+                buffer_row: buffer_row.buffer_row,
```

Comment:
When displaying the absolute line number for the current, wrapped line, we need to know what that line number is, somehow. Storing it in the wrapped line row info lets us access it easily. However, it also means that it'll count relative line numbers for all wrapped lines regardless of your setting (because we check if `buffer_row.is_some()`). If there was a way to check the editor setting here, then we could set it conditionally, but I couldn't find any other references to editor in this code, so I don't think that's a good approach.